align_system/algorithms/kaleido_adm.py

```diff
@@ -276,25 +309,75 @@ def choose_action(self, scenario_state, available_actions, alignment_target, **k
             decision_environment=scenario_state.environment.decision_environment.unstructured.strip(),
             characters_str=characters_str)
 
-        # Scaling KDMA values by 10 (range should be 0-10)
-        if not isinstance(alignment_target, dict):
-            alignment_target = alignment_target.to_dict()
+        choices_unstructured = [a.unstructured for a in available_actions]
```

Comment:
Do we need to ensure these choices are unique like we do for Outlines?

Reply:
Ah good catch, yes I think that makes sense.  Looks like just using the outlines_adm `format_choices` function should cover it yes?

Reply:
I think so